addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/handler/live/TelldusLiveDeviceController.java

```diff
@@ -0,0 +1,346 @@
+import org.openhab.binding.tellstick.handler.live.xml.TellstickNetDevice;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.tellstick.JNA;
+import org.tellstick.device.TellstickDevice;
+import org.tellstick.device.TellstickDeviceEvent;
+import org.tellstick.device.TellstickException;
+import org.tellstick.device.TellstickSensorEvent;
+import org.tellstick.device.iface.Device;
+import org.tellstick.device.iface.DeviceChangeListener;
+import org.tellstick.device.iface.SensorListener;
+import org.tellstick.device.iface.SwitchableDevice;
+
+/**
+ * {@link TelldusLiveDeviceController} is the communication with Telldus Live service (Tellstick.NET and ZNET)
+ * This controller uses XML based Rest API to communicate with Telldus Live.
+ *
+ * @author Jarle Hjortland
+ */
+
+public class TelldusLiveDeviceController implements DeviceChangeListener, SensorListener, TelldusDeviceController {
+    private static final Logger logger = LoggerFactory.getLogger(TelldusLiveDeviceController.class);
+    private long lastSend = 0;
+    long resendInterval = 100;
+    public static final long DEFAULT_INTERVAL_BETWEEN_SEND = 250;
+    static final int REQUEST_TIMEOUT_MS = 5000;
+    AsyncHttpClient client;
+    static final String HTTP_API_TELLDUS_COM_XML = "http://api.telldus.com/xml/";
+    static final String HTTP_TELLDUS_CLIENTS = HTTP_API_TELLDUS_COM_XML + "clients/list";
+    static final String HTTP_TELLDUS_DEVICES = HTTP_API_TELLDUS_COM_XML + "devices/list?supportedMethods=19";
+    static final String HTTP_TELLDUS_SENSORS = HTTP_API_TELLDUS_COM_XML + "sensors/list?includeValues=1&includeScale=1";
+    static final String HTTP_TELLDUS_SENSOR_INFO = HTTP_API_TELLDUS_COM_XML + "sensor/info";
+    static final String HTTP_TELLDUS_DEVICE_DIM = HTTP_API_TELLDUS_COM_XML + "device/dim?id=%d&level=%d";
+    static final String HTTP_TELLDUS_DEVICE_TURNOFF = HTTP_API_TELLDUS_COM_XML + "device/turnOff?id=%d";
+    static final String HTTP_TELLDUS_DEVICE_TURNON = HTTP_API_TELLDUS_COM_XML + "device/turnOn?id=%d";
+
+    public TelldusLiveDeviceController() {
+    }
+
+    @Override
+    public void dispose() {
+        try {
+            client.close();
+        } catch (Exception e) {
+            logger.error("Failed to close client", e);
+        }
+    }
+
+    void connectHttpClient(String publicKey, String privateKey, String token, String tokenSecret) {
+        ConsumerKey consumer = new ConsumerKey(publicKey, privateKey);
+        RequestToken user = new RequestToken(token, tokenSecret);
+        OAuthSignatureCalculator calc = new OAuthSignatureCalculator(consumer, user);
+        this.client = new DefaultAsyncHttpClient(createAsyncHttpClientConfig());
+        try {
+            this.client.setSignatureCalculator(calc);
+            Response response = client.prepareGet(HTTP_TELLDUS_CLIENTS).execute().get();
+            logger.info("Response " + response.getResponseBody() + " tt " + response.getStatusText());

```

Comment:
reduce to debug and change to parameterized logging instead of string concatenation